### PR TITLE
Fixes after review

### PR DESCRIPTION
**Pull request by dirkmoors**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -6,8 +6,8 @@ release = register clean --all sdist bdist_wheel
 publish = register clean --all sdist bdist_wheel upload
 
 [flake8]
-max-line-length = 140
-exclude = tests/*,*/migrations/*,*/south_migrations/*
+max-line-length = 120
+exclude = tests/*,*/migrations/*,*/south_migrations/*,setup.py,build/*,*/compat.py
```
Comment (jgeskens) [inline setup.cfg]: You should exclude tests, because otherwise they stay ugly ;-)

File: src/kong/__init__.py [hunk 2]
```diff
@@ -1,8 +1,25 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals, print_function
 
+"""
```
Comment (jgeskens) [inline src/kong/__init__.py]: What we meant is that this should be somewhere in the readme, along the lines of "PyOpenSSL support, will automatically inject into urllib3". So you advertise it as a "feature", but at the same time people are warned :P

File: src/kong/client.py [hunk 3]
```diff
@@ -15,6 +15,14 @@
 from .compat import OK, CREATED, NO_CONTENT, NOT_FOUND, CONFLICT, INTERNAL_SERVER_ERROR, urljoin
 from .exceptions import ConflictError, ServerError
 
+########################################################################################################################
```
Comment (jgeskens) [inline src/kong/client.py]: Haha, this reminds me of: https://github.com/vikingco/p4-proxy/blob/master/p4_proxy/client.py#L75

File: src/kong/client.py [hunk 4]
```diff
@@ -15,6 +15,14 @@
...
+# BEGIN: CI fixes
+#
+#   Because of memory/performance limitations in the CI, it often happened that connections to Kong got messed up
+#   during unittests. To prevent this from happening, we've implemented both throttling and connection dropping as
+#   optional measures during testing.
+########################################################################################################################
+
 KONG_MINIMUM_REQUEST_INTERVAL = float(os.getenv('KONG_MINIMUM_REQUEST_INTERVAL', 0))
```
Comment (jgeskens) [inline src/kong/client.py]: What is the unit of the interval? Add a small note here.

File: setup.cfg (hunk 1 above)
Comment (dirkmoors) [inline setup.cfg]: Fixed

File: src/kong/__init__.py (hunk 2 above)
Comment (dirkmoors) [inline src/kong/__init__.py]: When the documentation will be created/extended, I'll keep in mind to add a section for this warning

File: src/kong/client.py (hunk 4 above)
Comment (dirkmoors) [inline src/kong/client.py]: Fixed

**Pull request merged**